src/lib/Output/Normalizer/JsonSerializableNormalizer.php

```diff
@@ -20,9 +20,11 @@ final class JsonSerializableNormalizer implements NormalizerInterface, Normalize
     /**
      * @param array<mixed> $context
      *
+     * @return string|array<mixed>
+     *
      * {@inheritDoc}
      */
-    public function normalize($object, ?string $format = null, array $context = []): string
+    public function normalize($object, ?string $format = null, array $context = []): string|array
```

Comment:
Actually, we should remove the return type, as [\JsonSerializable::jsonSerialize()](https://www.php.net/manual/en/jsonserializable.jsonserialize.php) method does not offer any guaranties about the returned type :disappointed: 

```suggestion
    public function normalize($object, ?string $format = null, array $context = [])
```